checkout-core/src/main/java/com/adyen/checkout/core/log/LogUtil.kt

```diff
@@ -53,8 +53,7 @@ object LogUtil {
     private val callerClassName: String
         get() {
             val stElements = Thread.currentThread().stackTrace
-            for (i in 1 until stElements.size) {
-                val ste = stElements[i]
+            for (ste in stElements.drop(1)) {
```

Comment:
Can `drop(1)` throw an exception if the list is empty or something?

Reply:
Nope, just returns empty list.